Assets/Scripts/Noise/Noise.cs

```diff
@@ -1,13 +1,15 @@
 using UnityEngine;
 
-/// <summary>
-/// Noise function to generate noise values for a three dimensional space.
-/// </summary>
-public interface Noise {
+namespace Noise {
     /// <summary>
-    /// Will return a noise value between [0.0, 1.0] for a given (X,Y,Z) coordinate.
+    /// Noise function to generate noise values for a three dimensional space.
     /// </summary>
-    /// <param name="position">Position in (X,Y,Z) space</param>
-    /// <returns>The noise value at that coordinate, a float between [0.0, 1.0]</returns>
-    float GetNoise(Vector3 position);
-}
\ No newline at end of file
+    public interface Noise {
```

Comment:
Interfaces must be prefaced with a capital "i". This should be INoise, and the file name should be as well

Reply:
Unless you want Noise to be a base class then use the ```public abstract class Noise```

Reply:
Cool, I changed this and HeightMap